src/ac_training_lab/picam/README.md

```diff
@@ -25,26 +25,40 @@ The following components are required for the equipment monitoring setup:
 - [MicroUSB Power Adapter (International Plugs)](https://www.pishop.ca/product/microusb-power-adapter-international-plugs/) - $12.95 CAD **[INTERNATIONAL OPTION]**
 - *Note: Pi Zero 2W uses micro USB for power, not USB-C*
 
+### Optional Troubleshooting Items
+These items are optional and don't factor into the final price shown. They're useful if you're unable to SSH into the machine:
+- [6FT High Speed HDMI Cable With Ethernet](https://www.pishop.ca/product/6ft-high-speed-hdmi-cable-with-ethernet-18gbps-28awg-gold-plated/) - $2.95 CAD **[OPTIONAL]**
+- [USB OTG Host Cable - MicroB OTG male to A female](https://www.pishop.ca/product/usb-otg-host-cable-microb-otg-male-to-a-female/) - $3.45 CAD **[OPTIONAL]**
+- [Wireless Keyboard and Mouse Combo with USB Dongle](https://a.co/d/dxV6WWj) - Amazon **[OPTIONAL]**
+- External Monitor **[OPTIONAL]**
+- *Note: These items provide a simple fix for situations where SSH access is unavailable*
+
 ### Mounting Hardware
 **Option A:** Off-the-shelf mount kit
 - [Pro Mini Camera Mount for Raspberry Pi Zero](https://www.pishop.ca/product/pro-mini-camera-mount-for-raspberry-pi-zero/) - $7.95 CAD
 - *Note: Off-the-shelf mount works but might need to be secured so the two laser cut pieces don't wobble relative to each other*
 
 **Option B:** DIY mounting solution (recommended):
 - [3D Printable PiCam Mount](https://github.com/AccelerationConsortium/ac-training-lab/tree/main/src/ac_training_lab/picam/_design) (STL files in repository)
-- [M2 Nylon Hex Standoff Spacer Screw Nut Assortment Kit (150 Pieces)](https://www.pishop.ca/product/m2-nylon-hex-standoff-spacer-screw-nut-assortment-kit-160-pieces/) - $7.95 CAD
-  - Alternative: M2 and M2.5 hardware from [McMaster-Carr](https://www.mcmaster.com/) as backup
+- **M2 Hardware (4 sets needed):**
+  - [M2 x 12mm Socket Head Cap Screws](https://www.mcmaster.com/91292A831/) - McMaster-Carr
+  - [M2 x 0.4mm Hex Nuts](https://www.mcmaster.com/90591A111/) - McMaster-Carr
+- **M2.5 Hardware (4 sets needed):**
+  - [M2.5 x 8mm Socket Head Cap Screws](https://www.mcmaster.com/91292A014/) - McMaster-Carr
+  - [M2.5 x 0.45mm Hex Nuts](https://www.mcmaster.com/90591A113/) - McMaster-Carr
+  - [M2.5 x 8mm Hex Standoffs](https://www.mcmaster.com/94669A120/) - McMaster-Carr
+- *Note: Assembly instructions available at [ThePiHut](https://thepihut.com/blogs/raspberry-pi-tutorials/pro-mini-camera-mount-assembly-guide)*
 
 ### Rod Clamp Assembly & Mounting
 - [Camera Desk Mount Table Stand](https://www.primecables.ca/p-407778-cab-lsd01-1s-camera-desk-mount-table-stand) - Prime Cables (~$20-30 CAD)
-- [5/16" Hex Nut](https://www.mcmaster.com/7648A72/) - McMaster-Carr for secure rod mounting (~$5-10 CAD)
+- [5/16" Hex Nut](https://www.mcmaster.com/product/91078A029) - McMaster-Carr for secure rod mounting (~$5-10 CAD)
 
 ### Tools Required (separate from total cost)
 - [Small Precision Screwdriver](https://www.mcmaster.com/7845A36/) - McMaster-Carr (~$15-20 CAD)
```

Comment:
Bad link. Here's one to a set - not sure which size is actually needed, but at least one from this set should do it

https://www.mcmaster.com/product/52985A22